OBF Bylaws.md

```diff
@@ -83,19 +85,19 @@ ARTICLE IV - NOMINATIONS AND ELECTIONS
+    a.	Voting for the Board of Directors shall be by electronic or paper ballot. The President or Vice President shall prepare the ballot of candidates to be used for the purpose of voting.
     b.	Voting shall be by Approval Vote. All nominees will be on the ballot, and each Director or member may cast as many votes as nominees, but at most one vote for each nominee. A nominee must receive a majority of votes (or 'Yes' votes) to win a seat. If there are more than one nominees for an office, or more nominees than seats to fill, the nominee or nominees with the highest number of votes (or 'Yes' votes) win the seats to be filled.
     c.	Only the three Officers (see Article II, Items 2 and 8) may be elected by the general membership. The Board Members at Large shall be elected by the Board of Directors.
     d.	Ties in elections shall be broken by the President.
     e.	Ballots submitted to the general membership shall be secret. Board of Director votes for Directors shall be secret.
-4.	If a current Board Member at Large is nominated and elected officer, her previous Board seat becomes vacant and therefore available for election, unless the Board also decides to reduce the size of the Board.
+4.	If a current Board Member at Large is nominated and elected officer, their previous Board seat becomes vacant and therefore available for election, unless the Board also decides to reduce the size of the Board.
 5.	Members of the Board of Directors are eligible to vote in their role as part of the general membership in elections submitted to the general membership. In such elections, each director has as many votes as any other member.
 6.	Qualifications
     a.	Nominees or appointees for Director positions (Board Members at Large) must be voting members of the OBF in good standing.
     b.	Nominees or appointees for Officer positions must have been voting members of the OBF for a minimum of one year, be in good standing, and have previously served on the Board of Directors for a minimum of one year.
     c.	The Board of Directors may waive qualification requirements at its discretion by unanimous consent of attending Directors, provided the quorum is met. If the membership requirement is waived for a nominee, the nominee automatically becomes a member upon election, and must meet all requirements to be in good standing within 4 (four) weeks after election. 
 7.	_Removed_.
-8.	Paper ballots shall be counted by 2 (two) active members of the OBF selected by the President. Electronic ballots will be counted electronically.
+8.	Paper ballots shall be counted during the meeting by 2 (two) active members of the OBF selected by the President. Electronic ballots will be counted electronically once each ballot is closed, which may be up to 24 hours later to facilitate participation by absentee board members in other time zones (who would not be counted toward quorum).
```

Comment:
One might argue the old wording is sufficiently broad that we don't need this change?

Reply:
Hilmar agrees on #101 that the old policy wording allows this anyway, and this only restricts us. I'll revert it...